src/components/mobile-menu-container/index.tsx

```diff
@@ -41,7 +44,10 @@ const MOBILE_MENU_MOTION = {
  * Handles rendering the Sign In and Sign Up UI elements in mobile viewports.
  * Intended to be used alongside `MobileMenuContainer`.
  */
-const MobileAuthenticationControls = () => {
+
+const MobileAuthenticationControls = ({
+	className,
+}: MobileAuthenticationControlsProps) => {
```

Comment:
If you change it to this, then you can revert the changes to `src/components/mobile-menu-container/types.ts`. You only need to add `HTMLAttributes` to the line importing from react, e.g. `import { ForwardedRef, forwardRef, HTMLAttributes } from 'react'`

```suggestion
}: HTMLAttributes<HTMLDivElement>) => {
```

Reply:
Awesome suggestion! 🎉 I updated the types in [this commit](https://github.com/hashicorp/dev-portal/pull/2553/commits/05a1eb4741dc35e7602004a8aca13d2deb2202cf)